ci/tag.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# This script is used by travis to auto tag our releases
+
+# Configure git
```

Comment:
If we want to do more build stuff w/ git then this section should be moved to another script file and run during `before_deploy`